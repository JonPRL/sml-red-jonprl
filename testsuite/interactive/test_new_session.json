{"cmd": "new_session", "name": "test_session"}
