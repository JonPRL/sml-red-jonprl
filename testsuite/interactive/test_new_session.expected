{"session_id": "test_session"}
